apps/extension/src/ui/hooks/useChains.tsx

```diff
@@ -7,27 +9,35 @@ import {
 import { useMemo } from "react"
 import { useRecoilValue } from "recoil"
 
-export const useChains = (withTestnets: boolean) => {
-  // // keep db data ption("chains")
+type ChainsFilter = "all" | "enabledWithTestnets" | "enabledWithoutTestnets"
```

Comment:
Prefer to use Enums or constants for values like this
Although, since we now have two states (all/enabled and testnets on/off) perhaps an options object would be more suitable?

Reply:
done in asset discovery pr